clang/docs/checkedc/Setup-and-Build.md

```diff
@@ -57,138 +62,223 @@ need to end with line feeds. Visual Studio preserves line endings for files, so
-```
-
-## Setting up a build directory
-
-1. LLVM and Clang use CMake, which is a meta-build system generator. It generates build systems for a specific platform.
-2. Create a build directory that is a sibling of your llvm source tree.  For example, if your sources are in MyDir\src, create MyDir\build.      
-3. Be sure to exclude the build directory from anti-virus scanning.   On Windows 10, go to Settings->Update & Security->Windows Defender->Add an exclusion.
-4. CMake will produce a build system by default that builds code generators for all LLVM-supported architectures.
-   This can increase build and link times.  You might want to build the code generator for a specific target, such as x86.  To
-   do that,  add `-DLLVM_TARGETS_TO_BUILD="X86"` to the command-line below.
-4. Make sure to set the following CMake flag to enable clang in your builds: -DLLVM_ENABLE_PROJECTS=clang
-5. Make sure that you are using whatever shell you normally do compiles in.
-
-On Windows and Linux, cd your build directory and invoke CMake:
-```
-  cmake -G Ninja {llvm-path}
-```
-where `{llvm-path}` is the path to the root of your LLVM repo.
-
-### Building an LLVM package (advanced topic)
-If you are just trying out Checked C, you can safely ignore this section.  If
-you plan to build an LLVM package for installation on other machines,  we
-recommend that you build a release build of clang with assertions on and only
-include the toolchain in
-the package.  On Windows, you can add the following flags to your CMake options:
-```
-  -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_INSTALL_TOOLCHAIN_ONLY=ON -DLLVM_USE_CRT_RELEASE=MT
-```
-On UNIX you can add,
-```
-  -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON -DLLVM_INSTALL_TOOLCHAIN_ONLY=ON
-```
-
-## Building
-
-Note that the first time that you build clang, it may take over an hour to
-build.  This is because LLVM is being built.   The debug build of LLVM is
-particularly slow because it bottlenecks on table generation. LLVM generates
-architecture-specific tables at build time that are used during code
-generation.  The default table generation algorithm is very slow in debug
-builds.  Subsequent builds during development will be much faster (minutes, not
-an hour).
-
-### On UNIX
-Change to your build directory and build `clang`:
-```
-  ninja clang
-```
-
-### On Windows
-
-For day-to-day development, we recommend building from Visual Studio.  This
-will improve your productivity significantly because it will give you all the
+## Checkout and Build Instructions for Checked C Compiler
+
+You will need to choose a drive that has at least 50 Gbytes free. You may need
+lots of space for the sources and the build. You can store the sources in any
+directory that you want. You should avoid spaces in parent directory names
+because this can confuse some tools.
+
+### Instructions for Linux:
+
+1. Choose any directory as your working directory. We will refer to to this
+directory as \<WORK_DIR\>.
```

Comment:
Linux uses forward slashes. I guess we can leave out the slashes around `WORK_DIR`.